plugin-core/src/main/java/com/salesforce/bazel/eclipse/projectview/ProjectViewEditor.java

```diff
@@ -104,6 +104,12 @@ protected void editorSaved() {
         List<BazelPackageLocation> packageLocations = new ArrayList<>(projects.length);
         for (IJavaProject project : projects) {
             // get the target to get at the package path
+            List<String> targets = BazelEclipseProjectSupport.getBazelTargetsForEclipseProject(project.getProject(), false);
+            if (targets == null || targets.isEmpty()) {
+                // this shouldn't happen, but if it does, we do not want to blow up here
+                // instead return null to force re-import
+                return null;
+            }
```

Comment:
I ran into a situation where the inner Eclipse was in a bad state, and targets was null...